ui/admin/app/templates/scopes/scope/aliases/index.hbs

```diff
@@ -18,155 +18,181 @@
     {{#if (can 'create model' this.scope collection='aliases')}}
-      {{#if @model}}
-        <Hds::Button
-          @route='scopes.scope.aliases.new'
-          @text={{t 'resources.alias.titles.new'}}
-        />
-      {{/if}}
+      <Hds::Button
+        @route='scopes.scope.aliases.new'
+        @text={{t 'resources.alias.titles.new'}}
+      />
     {{/if}}
   </page.actions>
-  <page.body>
 
-    {{#if @model}}
-      <Hds::Table
-        @model={{@model}}
-        @columns={{array
-          (hash label=(t 'resources.alias.title'))
-          (hash label=(t 'resources.alias.titles.points_to'))
-          (hash label=(t 'resources.scope.title'))
-          (hash label=(t 'resources.alias.titles.resource_id'))
-          (hash label=(t 'titles.actions') isVisuallyHidden=true)
-        }}
-        @valign='middle'
-      >
-        <:body as |B|>
-          <B.Tr>
-            <B.Td>
-              {{#if (can 'read model' B.data.alias)}}
-                <LinkTo
-                  @route='scopes.scope.aliases.alias'
-                  @model={{B.data.alias.id}}
-                >
-                  {{B.data.alias.value}}
-                </LinkTo>
-              {{else}}
-                <Hds::Text::Body @tag='p'>
-                  {{B.data.alias.value}}
-                </Hds::Text::Body>
-              {{/if}}
-              {{#if B.data.alias.description}}
-                <Hds::Text::Body @tag='p'>
-                  {{B.data.alias.description}}
-                </Hds::Text::Body>
-              {{/if}}
-            </B.Td>
-            <B.Td>
-              {{#if B.data.target.displayName}}
-                <div class='aliases-list-cell-item'>
-                  <span><FlightIcon @name='target' /></span>
-                  <span> {{B.data.target.displayName}} </span>
-                </div>
-              {{else}}
-                <Hds::Text::Body
-                  @tag='p'
-                  @size='100'
-                  @weight='medium'
-                  @color='faint'
-                >
-                  {{t 'titles.not-set'}}
-                </Hds::Text::Body>
-              {{/if}}
-            </B.Td>
-            <B.Td>
-              <Hds::Badge @text={{t 'titles.global'}} @icon='globe' />
-            </B.Td>
-            <B.Td>
-              {{#if B.data.alias.destination_id}}
-                <Hds::Copy::Snippet
-                  @textToCopy={{B.data.alias.destination_id}}
-                  @color='secondary'
-                />
-              {{/if}}
-            </B.Td>
-            <B.Td>
-              {{#let
-                (hash
-                  canUpdate=(can 'update model' B.data.alias)
-                  canDelete=(can 'delete model' B.data.alias)
-                )
-                as |perms|
-              }}
-                {{#if (or perms.canUpdate perms.canDelete)}}
-                  <Hds::Dropdown as |dd|>
+  <page.body class='search-filtering'>
+    {{#if @model.aliasesExist}}
```

Comment:
Hmm, can we name this differently? right now, `@model.aliases` and `@model.aliasesExist` read the same to me. Naming it with more context would tell us why we can't use `@model.aliases` to toggle the search bar visibility

Reply:
Would something like `@model.doAliasesExist` read better? Try to make it more clear that this is a boolean compared to an array of alias models.

Reply:
I will say we have been using this naming pattern across other pages so if we want to change it, we should apply it everywhere.

Reply:
good points, i think `doAliasesExist` is clear. Ill also update in other places since its a simple change.